contrib/gdb_debugging_scripts/thread_tran.gdb

```diff
@@ -32,7 +32,7 @@ define thread_find_by_tid
   set $i = 0
   set $arg1 = 0
   while $i < thread_Manager.num_total
-    if thread_Manager.thread_array[$i]->tid == $arg0
+    if thread_Manager.thread_array[$i]->get_posix_id () == $arg0
```

Comment:
I don't think it works for debugging a crash file. Since majority of our debugging job is working with a core file, I'd like to propose to keep a public member. This makes our life easier.

Reply:
I'll try to address this issue